float32 height_local
float32 width_local
float32 quality_local
float32[] center_local
float32[] growth_center_local
float32 tilt_local
float32[] corners_local
float32[] init_pos_global